WordPress/Classes/Models/ReaderCrossPostMeta.swift

```diff
@@ -1,8 +1,10 @@
 import Foundation
 import CoreData
 
-@objc public class ReaderCrossPostMeta : NSManagedObject
+@objc public class ReaderCrossPostMeta : NSManagedObject, ManagedObject
```

Comment:
For consistency, maybe implement the protocol as an extension in the test like the others models.  Alternatively, update the other models to implement the protocol directly.